Code completion in completionvalues1.xml for <item name="android:alwaysDrawnWithCache">^  false  </item> selecting true:
@@ -13 +13
-         <item name="android:alwaysDrawnWithCache">^  false  </item>
+         <item name="android:alwaysDrawnWithCache">true^</item>
